config.yml.erb

```diff
@@ -323,7 +323,7 @@ dcdo_table_name:       dcdo_<%=env%>
 gatekeeper_table_name: gatekeeper_<%=env%>
 
 # Poste (internal transactional email processing framework)
-poste_host: <%= code_org_url.sub('//', '') %>
+poste_host: '<%= code_org_url.sub('//', '') %>'
```

Comment:
The issue is that now this returns `localhost.code.org:3000` on CI servers. The YAML parser sees the `:` and says "It looks like you are trying to define a Hash... oops looks like your Hash syntax is bad... \<throws exception\>". By quoting it, we explicitly set the type for the value rather than relying on the YAML parser to divine it.

Before, the CI servers would incorrectly set this to `test.code.org`.

Reply:
Ahh, that makes sense. Oh, yaml :upside_down_face: 

Thanks!